external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -115,66 +120,560 @@ To reuse or extend existing dimensions, attributes, units, or measures, provide
-| `label`             | The title of the column (Optional)                                                                                                                                                                                   | The capital case of the header in the csv file with spaces replacing underscores |
-| `description`       | A description of the contents of the column (Optional)                                                                                                                                                               | *none*                                                                           |
-| `from_existing`     | The uri of the resource for reuse/extension (Optional)                                                                                                                                                               | *none*                                                                           |
-| `definition_uri`    | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional)                                                                                            | *none*                                                                           |
-| `uri_override`      | Override the uri created automatically for the column (Optional) (Advanced)                                                                                                                                          | `tidy_data.csv#uri_safe_column_header_from_csv`                                  |
-| `cell_uri_template` | Override the uri generated for values within the uri (Optional) (Advanced)                                                                                                                                           | *none*                                                                           |
-| `code_list`         | Link to an existing code list (uri), supress a code-list (false), file path to a [code-list-config.json](code-list-config.md) (uri), [in-line code list](code-list-config.md) (json), or generate a code-list (true) | true                                                                             |
+Note that the `type` of this column has not been defined, meaning that csvcubed will interpret the `Year` column as a dimension by default.
+
+```json
+{ ...
+   "columns": {
+      "Entrant": {
+         "type": "dimension",
+         "description": "The act representing Sweden at Eurovision for the given year",
+         "definition_uri": "http://example.org/swedish-eurovision-acts",
+         "code_list": "entrant_code_list_config.json"
+      }
+   }
+}
+```
+
+The `Entrant` column has been explicitly configured with a `type` of dimension, although strictly speaking this is unnecessary, as a column will be designated as a dimension by default if `type` is not defined.
```

Comment:
Not sure we need to mention the default behaviour of `type` every time we talk about these definitions. Seems a bit repetetive.